HW_1/task_4.py

```diff
@@ -0,0 +1,20 @@
+
+# Задача 8: Требуется определить, можно ли от шоколадки
+# размером n × m долек отломить k долек, если разрешается
+# сделать один разлом по прямой между дольками
+# (то есть разломить шоколадку на два прямоугольника).
+
+# 3 2 4 -> yes
+# 3 2 1 -> no
+
+
+n = int(input('Введите размер шоколадки: '))
+m = int(input('Введите размер одной дольки:'))
+k = int(input('Введите размер другой дольки:'))
+
+if n * m < k or k == n * m:
```

Comment:
Мне кажется, или эти условия можно объединить так
```suggestion
if n * m <= k:
```